apps/patient/src/components/BrandedPharmacyCard.tsx

```diff
@@ -1,56 +1,68 @@
 
-import capsuleLogo from '../assets/capsule_logo.png';
-import amazonPharmacyLogo from '../assets/amazon_pharmacy.png';
+import capsuleLogo from '../assets/capsule_logo_small.png';
+import amazonPharmacyLogo from '../assets/amazon_pharmacy_logo_small.png';
 import altoLogo from '../assets/alto_logo.svg';
-import costcoLogo from '../assets/costco_pharmacy_logo.png';
+import costcoLogo from '../assets/costco_logo_small.png';
 
 import capsulePharmacyIdLookup from '../data/capsulePharmacyIds.json';
+import { PharmacyInfo } from './PharmacyInfo';
 
 interface Props {
   pharmacyId: string;
+  selected: boolean;
   handleSelect?: ((id: string) => void) | undefined;
 }
 
-const PHARMACY_BRANDING = {
+export const PHARMACY_BRANDING = {
   [process.env.REACT_APP_AMAZON_PHARMACY_ID as string]: {
     logo: amazonPharmacyLogo,
+    name: 'Amazon Pharmacy',
     description: 'Delivers in 2-5 days'
   },
   [process.env.REACT_APP_ALTO_PHARMACY_ID as string]: {
     logo: altoLogo,
+    name: 'Alto Pharmacy',
     description: 'Delivers as soon as today'
   },
   [process.env.REACT_APP_COSTCO_PHARMACY_ID as string]: {
     logo: costcoLogo,
+    name: 'Costco Pharmacy',
     description: 'Delivers in 1-2 days'
-  }
+  },
+  ...Object.fromEntries(
+    Object.keys(capsulePharmacyIdLookup).map((id) => [
+      id,
+      {
+        logo: capsuleLogo,
+        name: 'Capsule Pharmacy',
+        description: 'Next day delivery'
+      }
+    ])
+  )
 };
-// TODO: need to make this more elegant
-const capsulePharmacyIds = Object.keys(capsulePharmacyIdLookup);
-for (let i = 0; i < capsulePharmacyIds.length; i++) {
-  PHARMACY_BRANDING[capsulePharmacyIds[i]] = {
-    logo: capsuleLogo,
-    description: 'Delivers as soon as today'
-  };
-}
 
-export const BrandedPharmacyCard = ({ pharmacyId, handleSelect }: Props) => {
+export const BrandedPharmacyCard = ({ pharmacyId, selected, handleSelect }: Props) => {
   const brand = PHARMACY_BRANDING[pharmacyId];
   if (!brand) return null;
 
-  const tagline = brand.description ? (
-    <Text fontSize="sm" color="gray.500">
-      {brand.description}
-    </Text>
-  ) : null;
+  const pharmacy = { id: pharmacyId, name: brand.name, logo: brand.logo };
+
+  const selectable = !!handleSelect;
 
   return (
-    <Box bgColor="white" w="full" cursor="pointer" onClick={() => handleSelect?.(pharmacyId)}>
-      <VStack align="start" spacing={2}>
-        <Image src={brand.logo} width="auto" height="30px" />
-        {tagline}
-      </VStack>
-    </Box>
+    <Card
+      bgColor="white"
+      border="2px solid"
+      borderColor={selected && selectable ? 'brand.500' : 'white'}
+      borderRadius="lg"
+      onClick={() => selectable && handleSelect(pharmacyId)}
+      mx={{ base: -3, md: undefined }}
+      cursor={selectable ? 'pointer' : undefined}
+    >
+      <CardBody p={3}>
+        <PharmacyInfo pharmacy={pharmacy} tagline={brand.description} boldPharmacyName={false} />
```

Comment:
I'm kinda surprised this even typechecks. I guess its because we have a bunch of optional types. 

Its going to be annoying later than we're only providing `id` `name` and `logo` to this component. 

But 🤷 , deal with it later

Reply:
Specifically here I'd do one of two things:
- make `PharmacyInfo` accept a `id`, `name` and `logo` (instead of `pharmacy`). That way you know those are the only pieces you have.
- Make `BrandedPharmacyCard` accept a `Pharmacy` rather than `pharmacyId`. Then pass in the pharmacy to `PharmacyInfo`

Reply:
Yeah this is the pain of not having correct delivery pharmacy types in our db